peachjam/static/bootstrap/scss/_variables.scss

```diff
@@ -486,7 +486,7 @@ $grid-breakpoints: (
 $container-max-widths: (
   sm: 540px,
   md: 720px,
-  lg: 960px,
+  lg: 970px,
```

Comment:
This is going to get lost next time we upgrade bootstrap.

Is it critical we have this?

Reply:
No. it's not